stock_valued_picking_report/models/stock_pack_operation.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Pedro M. Baeza - Tecnativa <pedro.baeza@tecnativa.com>
+# Copyright 2015 Antonio Espinosa - Tecnativa <antonio.espinosa@tecnativa.com>
+# Copyright 2016 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models, api
+from openerp.addons.decimal_precision import decimal_precision as dp
+
+
+class StockPackOperation(models.Model):
+    _inherit = "stock.pack.operation"
+
+    sale_line = fields.Many2one(
+        comodel_name='sale.order.line',
+        compute='compute_sale_order_line_fields',
+        string="Related order line", readonly=True)
```

Comment:
`readonly=True` is implicit for computed fields that have no inverse method. You can remove that.